package.json

```diff
@@ -180,6 +180,15 @@
                     "type": "boolean",
                     "description": "%AWS.configuration.enableCodeLenses%",
                     "default": true
+                },
+                "aws.togglePrompts": {
+                    "type": "object",
```

Comment:
What if this were an array? This would remove a fair amount of the concerns regarding true/false/non-boolean typing. Might be a bit more understandable to the user, especially if this was renamed `aws.doNotShowPrompts` or something.

Reply:
I think you're suggestion of an array is great.  Originally I was thinking of mapping each prompt, but an array of disabled prompts is cleaner and is easier to append.  It eliminates any maintenance cost for future disable prompts.